packages/components/pie-webc-core/src/functions/customEventDispatcher.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Dispatch a custom event.
+ *
+ * To be used whenever we have behavioural events we want to
+ * bubble up through the component.
+ * @param {Element} element The element to dispatch the event from.
+ * @param {string} eventType
+ * @param {any} detail
+ */
+export function dispatchCustomEvent (element: Element, eventType: string, detail?: CustomEventInit['detail']) : void {
+    const event = new CustomEvent(eventType, {
+        bubbles: true,
+        composed: true,
```

Comment:
I am unsure if we should assume that every custom event should have `bubbles` and `composed` set to true. 

Are we sure this is worth extracting and reusing?

Reply:
Currently for all the use cases it is true. I think we will be able to upgrade it in future if there will be need for it